packages/paste-website/tsconfig.json

```diff
@@ -7,6 +7,6 @@
     "target": "esnext",
     "emitDeclarationOnly": false
   },
-  "include": ["../../package.json", "./src/**/*", "./types/**/*", "./static/**/*"],
+  "include": ["../../package.json", "./src/**/*", "./types/**/*", "./static/**/*", "./stories/**/*"],
```

Comment:
Why this change?

Reply:
Oh I had a story for a while but deleted it. It automagically appeared